files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -114,7 +114,7 @@ def update_filter_metadata(metadata_to_update):
 
 def get_client_token(client_id: str, client_secret: str):
     try:
-        token_url = f"http://revproxy-service/user/oauth2/token"
+        token_url = "http://revproxy-service/user/oauth2/token"
```

Comment:
Can we change this to `https`? Snyk is throwing this url as an error. I know that this is hitting reverse proxy but I think that https will still probably work.

Reply:
Yeah I wasn't sure if https would work, I was going to tell security this is a false-positive. But I can definitely test the https change

Reply:
well, actually, it doesn't look like https gonna work for revproxy 😕 https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/revproxy/revproxy-service.yaml#L8-L12

Reply:
Yeah, had a small chat with Jawad about this and it won't work as handling certs for all the services would be a huge issue we would have to maintain.